pkg/apiserver/handlers_test.go

```diff
@@ -200,6 +200,35 @@ func TestTimeout(t *testing.T) {
 	}
 }
 
+func TestHasApiPrefix(t *testing.T) {
```

Comment:
> Ok. there is no test for GetAttribs() yet, any hints on how to mock requestAttributeGetter. Are there factories for this or should I implement interfaces. A pointer to where to peek in the source code would be fine.

The code looks pretty tolerant of a stubbed `api.RequestContextMapper`.  I would stub the `APIRequestInfoResolver` as its done in `handlers_test.go`.